spec/system/comparisons/electricity_peak_kw_per_pupil_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'electricity_peak_kw_per_pupil' do
+  let(:school) { create(:school) }
+
+  before do
+    create(:advice_page, key: :electricity_intraday)
+
+    alert_run = create(:alert_generation_run, school: school)
+
+    baseload_alert = create(:alert_type, class_name: 'AlertElectricityPeakKWVersusBenchmark')
```

Comment:
concerned that if the class name is wrong here and the code then the test will pass when it shouldn't

Reply:
As a small improvement, what about adding a factory that accepts a Class rather than a name and returns an alert_type. That at least confirms that the class exists in the code base? 

Similarly for the alert we could add a factory or helper that confirms that an alert class exposes the variables we want to store the in the `Alert`. 

Provides a bit of consistency checking without us having to run every analysis?